src/content/configuration/stats.md

```diff
@@ -7,6 +7,13 @@ contributors:
   - jungomi
   - ldrick
   - jasonblanchard
+  - TeffenEllis
+  - jbottigliero
+  - sokra
+  - renjithvk
+  - pierreneter
+  - jasonblanchard
+  - Hoten
```

Comment:
Can you please also mention the source where you found missing contributors?
I see this on GitHub. 
![image](https://user-images.githubusercontent.com/5961873/38513398-c5d08356-3c4b-11e8-859c-024aaa2d552d.png)

Reply:
Currently here only added 4 contributors list:
![editing webpack js org stats md at master webpack webpack js org](https://user-images.githubusercontent.com/11629673/38535405-e57c8cfe-3ca0-11e8-8896-1b3a226f66af.png)

I saw there have 7 contributors:
![webpack js org stats md at master webpack webpack js org](https://user-images.githubusercontent.com/11629673/38535479-47e60b9a-3ca1-11e8-9aee-49746f690c4a.png)

That why updated the contributors' list

Reply:
I mean the total list is now of 11 people. :D

Reply:
@dhruvdutt I did fix that, thanks